definitions/features/upstream_repos.rb

```diff
@@ -1,40 +1,25 @@
-class Features::SystemRepos < ForemanMaintain::Feature
+class Features::UpstreamRepos < ForemanMaintain::Feature
   metadata do
-    label :system_repos
-    description 'Feature for operations on yum repositories of system'
+    label :upstream_repos
+    description 'Feature for upstream repositories'
```

Comment:
Feature for what? This somehow is missing a verb or something. Maybe "for managing upstream repositories" or the old "operations on"?